### DISCUSSION TITLE
想问问web备份怎么使用，方便出个攻略么，研究了半天，没备份上，希望能出一个退出APP自动备份或者自动读取最新备份的步骤

### DISCUSSION THREAD

**Discussion by heibai082** [xiaohucode/yidaRule#128] [Ideas]
想问问web备份怎么使用，方便出个攻略么，研究了半天，没备份上，希望能出一个退出APP自动备份或者自动读取最新备份的步骤

**Comment by xiaohucode** [comment]
参考坚果云帮助 https://help.jianguoyun.com/?p=2064

**Reply by heibai082** [reply]
您好，我这边点了备份，只有本地备份，但是webdav不会备份，只有自动备份可以上传，然后点恢复备份，没反应，是我这边操作有误么？我是想安卓和苹果平板进度同步而已

**Reply by xiaohucode** [reply]
上传webdav失败会提示错误。把账号和密码填写正确就行了
